src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -287,7 +287,7 @@ public void testBanditColdStartAction() {
 
     // Attempt to get a bandit assignment
     Optional<String> stringAssignment = EppoClient.getInstance().getStringAssignment(
-      "subject2",
+      "subject1",
```

Comment:
GitHub diff is a bit confused as the old "ColdStart" test was actually for the "Uninitialized" case. So that one has been adjusted, and this one added. Probably easiest to just look at the green and not look at the red part of this.